README.md

```diff
@@ -1,15 +1,203 @@
-# An alternative http library and web framework inspired by hyper
+<h1 align="center">Xitca-Web</h1>
+<p align="center">
+  <b align="center"><a href="README.md">Readme</a></b> |
+  <b><a href="https://github.com/HFQR/xitca-web">GitHub</a></b> |
+  <b><a href="https://docs.rs/xitca-web/latest/xitca_web/">Documentation</a></b>
+  <br /><br />
+  <a href="#">
+    <img
+      alt="GitHub code size in bytes"
+      src="https://img.shields.io/github/languages/code-size/HFQR/xitca-web?style=flat-square"
+    />
+  </a>
+  <a href=""
+    ><img
+      alt="Maintenance"
+      src="https://img.shields.io/maintenance/yes/2024?style=flat-square"
+    />
+  </a>
+  <br />
+  <br />
+  <i>
+    A HTTP library and web framework written in 100% safe Rust with zero-copy
+    serialization/deserilization support
+  </i>
+</p>
 
-## Minimum Supported Rust Version
-- 1.79 [^1]
+<br />
+<img src="./images/server.jpg" alt="An image of a server" align="center" />
```

Comment:
> I would prefer this image removed. It serves no obvious purpose and can take up some extra resource when cloning the repo. In some part of the world accessing github can be hard and slow and I know this in first hand experience.

No I just kept this image because I have seen many popular projects do it. Also, I think images can attract the reader quite a lot and grabs their attention. So that's why. Additionally for speed I think we can compress the image down if you wish this can speed things by a lot. What do you say? :slightly_smiling_face:

Reply:
I would still prefer leaving anything that are not functionally part of the project out. In some countries accessing github can be as slow as single digit kilobytes per second. An irrelevant image even compressed can waste people's valuable time and bandwidth.

There is also the issue that irrelevant stuff can pile up over time if we accept the practice. There is always more asset can be included just for the look.

Reply:
> I would still prefer leaving anything that are not functionally part of the project out. In some countries accessing github can be as slow as single digit kilobytes per second. An irrelevant image even compressed can waste people's valuable time and bandwidth.
> 
> There is also the issue that irrelevant stuff can pile up over time if we accept the practice. There is always more asset can be included just for the look.

I don't think it should be a major problem I mean even if the image does not load I think it is still readable though but I think having a blank readme again is quite bad like it is does not grab the reader's attention alot and it could make the project left out of significant users or contributors I feel. But again if you insist I will remove it :slightly_smiling_face: .0